src/site/content/en/blog/origin-trials/index.md

```diff
@@ -17,17 +17,21 @@ Origin trials enable you to build functionality your users can try for the
 duration of the trial. When Chrome offers an origin trial, you can register for
 the trial to enable the feature for all users on your
 [origin](https://web.dev/same-site-same-origin/#origin), without requiring your
-users to toggle any flags or switch to a different version of Chrome. Likewise,
-your developers can try out demos and prototypes. Origin trials also help Chrome
-engineers understand how new features are used, and how they may interact with
-other web technologies.
+users to toggle any flags or switch to an alternative build of Chrome (though
+they may need to upgrade). Likewise, your developers can try out demos and
+prototypes. Origin trials also help Chrome engineers understand how new features
+are used, and how they may interact with other web technologies.
 
 Origin trials are public and open to all developers, and limited in duration and
 usage. Additionally, origin trials are a self-managed process with limited
 documentation and support. Participants should be willing and able to work
-relatively independently using the documentation available.
+relatively independently using the documentation available, which, at this stage,
+will likely be limited to specs and explainers. Though, web.dev tries to provide
```

Comment:
I don't think beginning the sentence with 'Though'. 

Maybe

Reply:
Done.